src/styles/__cycle.scss

```diff
@@ -0,0 +1,20 @@
+@for $index from 1 through 31 {
+  .calendar__day--#{$index}::before {
+    content: "#{$index}";
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
+}
+
+@for $lenght from 28 through 31 {
+  .calendar--month-lenght--#{$lenght}
+  .calendar__day:nth-child(n + #{$lenght + 1}) {
+    display: none;
+  }
+}
+
+@each $day, $start in $start-days {
+  .calendar--start-day--#{$day} .calendar__day:first-child {
+    margin-left: ($box-size + $border-size) * $start;
+  }
+}
```

Comment:
better place all these cycles to calendar block and use nesting. it would be clearer to read